imports/plugins/core/payments/server/no-meteor/resolvers/Mutation/enablePaymentMethodForShop.js

```diff
@@ -0,0 +1,14 @@
+import { decodeShopOpaqueId } from "@reactioncommerce/reaction-graphql-xforms/shop";
+
+export default async function enablePaymentMethodForShop(parentResult, { input }, context) {
+  const { clientMutationId } = input;
+  input.shopId = decodeShopOpaqueId(input.shopId);
+  delete input.clientMutationId;
```

Comment:
I generally try to avoid mutating an argument unless it's a function designed to mutate. This could have unintended consequences. Can you make a new input object instead?